### DISCUSSION TITLE
Patient Not Available

### DISCUSSION THREAD

**Discussion by garyisaac** [StanfordSpezi/.github#43] [Challenges and Help]
### What Stanford Spezi module is your challenge related to?

Spezi

### Description

I have installed and started the app successfully. I followed the instructions for using SMART Launch. The app seems to launch but then hangs with the dashboard logo. I get the following error after refreshing the browser. I tried different patients and providers.

Uncaught runtime errors:

ERROR
Patient is not available
read@http://localhost:3000/static/js/bundle.js:28966:30
getCurrentPatient@http://localhost:3000/static/js/bundle.js:2523:112
./src/context/PatientContext.tsx/PatientProvider/<@http://localhost:3000/static/js/bundle.js:2526:5
commitHookEffectListMount@http://localhost:3000/static/js/bundle.js:74239:30
commitPassiveMountOnFiber@http://localhost:3000/static/js/bundle.js:75732:42
commitPassiveMountEffects_complete@http://localhost:3000/static/js/bundle.js:75704:38
commitPassiveMountEffects_begin@http://localhost:3000/static/js/bundle.js:75694:45
commitPassiveMountEffects@http://localhost:3000/static/js/bundle.js:75684:38
flushPassiveEffectsImpl@http://localhost:3000/static/js/bundle.js:77569:32
flushPassiveEffects@http://localhost:3000/static/js/bundle.js:77521:18
./node_modules/react-dom/cjs/react-dom.development.js/commitRootImpl/<@http://localhost:3000/static/js/bundle.js:77336:13
workLoop@http://localhost:3000/static/js/bundle.js:112724:46
flushWork@http://localhost:3000/static/js/bundle.js:112702:18
performWorkUntilDeadline@http://localhost:3000/static/js/bundle.js:112939:25

### Reproduction

Follow instructions on the Github repo page.

### Expected behavior

Expected the app to launch.

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
Hi @garyisaac, are you referring to the code at https://github.com/StanfordSpezi/SpeziSMARTonFHIR?
If so, @vishnuravi might be able to provide some insights there 👍

**Comment by vishnuravi** [comment]
Hi @garyisaac, sorry that you are having trouble with the [SpeziSMARTonFHIR](https://github.com/StanfordSpezi/SpeziSMARTonFHIR) application. We have tried to reproduce your issue with the [SMART Launcher](https://launch.smarthealthit.org/), but were not able to. 

Would you be able to share some more detail about the configuration options you chose in the launcher? You may share the URL that appears in your browser after you have entered in the options in the launcher, as the options are also encoded in the link itself.

**Comment by garyisaac** [comment]
[SpeziSMART.docx](https://github.com/StanfordSpezi/.github/files/13337501/SpeziSMART.docx)

Thank you for your help. I have attached a document showing my install and launch using SMART Launcher. The app hangs and the source states javascript must be enabled. My browser does enable browser.

Thank you!
